pex/bin/pex.py

```diff
@@ -500,9 +500,11 @@ def configure_clp_pex_environment(parser):
         callback=process_platform,
         help="The platform for which to build the PEX. This option can be passed multiple times "
         "to create a multi-platform pex. To use the platform corresponding to the current "
-        "interpreter you can pass `current`. To target any other platform you pass a string "
-        "composed of fields: <platform>-<python impl abbr>-<python version>-<abi>. These fields "
-        "stem from wheel name conventions as outlined in "
+        "interpreter you can pass `current` "
+        "(which is the default value if this option is not provided). "
```

Comment:
The parenthetical contradicts the default above and its not always true either. Consider the case where 1 interpreter is specified - say via `--python python2.7` and yet the Pex CLI is running under Python 3.6. In that case the resulting PEX file will only be built using a local Python 2.7 interpreter and not additionally for `current` which is a local Python 3.6 interpreter. How about just dropping this addition?

Reply:
Great!

Reply:
Thank you for explaining this.

Reply:
After reverting the change you noted, I modified this further to include the distinction you gave above:
> If any distributions need to be built, use the --python or --interpreter-constraint argument instead, providing the corresponding interpreter.